components/automate-ui/src/app/components/authorized/authorized.component.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('AuthorizedComponent', () => {
   let fixture: ComponentFixture<AuthorizedComponent>;
   let dispatchSpy: jasmine.Spy;
 
-  beforeEach(async(() => {
+  beforeEach(waitForAsync(() => {
```

Comment:
you'll see this all throughout the update.  `async` has been deprecated in favor of `waitForAsync`